### ISSUE TITLE
panic: mpv API error: option not found (-5)

### ISSUE THREAD

**Issue by logix2** [aykevl93/plaincast#9]
I'm getting this error trying to run plaincast on ubuntu 18.10, libmpv 0.29.0:

```
panic: mpv API error: option not found (-5)

goroutine 35 [running]:
github.com/aykevl/plaincast/apps/youtube/mp.(*MPV).checkError(0xc4201285c0, 0x7f22fffffffb)
	/home/logix/.go/src/github.com/aykevl/plaincast/apps/youtube/mp/mpv.go:343 +0x167
github.com/aykevl/plaincast/apps/youtube/mp.(*MPV).setOption(0xc4201285c0, 0x7d2d10, 0x12, 0xc400000003, 0xc420246048)
	/home/logix/.go/src/github.com/aykevl/plaincast/apps/youtube/mp/mpv.go:151 +0xa2
github.com/aykevl/plaincast/apps/youtube/mp.(*MPV).setOptionFlag(0xc4201285c0, 0x7d2d10, 0x12, 0x7e7d01)
	/home/logix/.go/src/github.com/aykevl/plaincast/apps/youtube/mp/mpv.go:129 +0x70
github.com/aykevl/plaincast/apps/youtube/mp.(*MPV).initialize(0xc4201285c0, 0xc4201285c0, 0xc420094600)
	/home/logix/.go/src/github.com/aykevl/plaincast/apps/youtube/mp/mpv.go:63 +0x10a
github.com/aykevl/plaincast/apps/youtube/mp.New(0xc4200944e0, 0x7e7d68)
	/home/logix/.go/src/github.com/aykevl/plaincast/apps/youtube/mp/player.go:26 +0xfa
github.com/aykevl/plaincast/apps/youtube.(*YouTube).init(0xc4200ee2c0, 0xc4201ac060, 0xc4200944e0)
	/home/logix/.go/src/github.com/aykevl/plaincast/apps/youtube/youtube.go:175 +0x1b8
github.com/aykevl/plaincast/apps/youtube.(*YouTube).run(0xc4200ee2c0, 0xc4201ac060)
	/home/logix/.go/src/github.com/aykevl/plaincast/apps/youtube/youtube.go:212 +0x159
created by github.com/aykevl/plaincast/apps/youtube.(*YouTube).start
	/home/logix/.go/src/github.com/aykevl/plaincast/apps/youtube/youtube.go:199 +0xcf
Traceback (most recent call last):
  File "<string>", line 4, in <module>
ImportError: No module named youtube_dl
```

Perhaps this is a too new libmpv version? What version would I need to get this to work?

**Comment by aykevl** [comment]
I'm using libmpv 0.23.0 from Debian stretch (Raspbian) and that one works.

Also, it looks like you don't have youtube-dl installed (or perhaps for the wrong Python version).

**Comment by aykevl** [comment]
I've found the bug, and probably fixed it in ba4a4fa82a8724c3e2d9bf9521de7cfb77866213 (in master). Let me know if this commit doesn't fix the issue ("option not found").
